runner/run.go

```diff
@@ -38,7 +39,7 @@ func Run(tests []test.FTWTest, c Config, out *output.Output) (*TestRunContext, e
 	}
 	client, err := ftwhttp.NewClient(conf)
 	if err != nil {
-		return &TestRunContext{}, err
+		return TestRunContext{}, err
```

Comment:
Out of curiosity: why did you change from pointer to value for the error?

Reply:
I don't remember, but let's revert the change until the reason is clear 😄